spec/requests/planning_application_list_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "API request to list planning applications", type: :request, show_exceptions: true do
+  let(:access_control_allow_origin) { response.headers['Access-Control-Allow-Origin'] }
+  let(:access_control_allow_methods) { response.headers['Access-Control-Allow-Methods'] }
+  let(:access_control_allow_headers) { response.headers['Access-Control-Allow-Headers'] }
+  let(:reviewer) { create :user, :reviewer }
+
+  before do
+    create(:decision, :granted, user: reviewer)
+  end
+
+  describe "format" do
```

Comment:
We could move the header variables under this `describe` block to make it a bit clearer where they're used. Having them at the top level makes it look a bit like they're relevant for all of the specs.